terraform/templates/defaults/validator_docker_compose.tpl

```diff
@@ -27,4 +27,4 @@ services:
       - "--alarm-names=${mem_alarm}"
       - "--alarm-names=${incoming_packets_alarm}"
       - "--cortex-instance-endpoint=${cortex_instance_endpoint}"
-
+      - "--rollup=${rollup}"
```

Comment:
This currently won't do anything. There's no variable defined for rollup in the terraform and it isn't being set when creating the validator.